app/index.html

```diff
@@ -24,6 +24,8 @@
   <script type="text/javascript" src="js/livebus/livebus.factory.js"></script>
 
   <script type="text/javascript" src="js/shared/shared.module.js"></script>
+  <script type="text/javascript" src="js/shared/ucBackground/ucBackground.module.js"></script>
+  <script type="text/javascript" src="js/shared/ucBackground/ucBackground.directive.js"></script>
```

Comment:
These changed are now pushed.

At the same time I made a mistake with the branches, so I pushed some other stuff too. More about that on a new comment.